packages/create-bison-app/template/package.json.ejs

```diff
@@ -42,8 +42,8 @@
   },
   "dependencies": {
     "@apollo/client": "^3.0.2",
-    "@chakra-ui/core": "^1.0.0-rc.5",
-    "@chakra-ui/theme": "^1.0.0-rc.5",
+    "@chakra-ui/react": "1.3.4",
```

Comment:
Latest version has build error on @chakra/popper dependency that this version does not have